homeassistant/components/speedtestdotnet/sensor.py

```diff
@@ -33,13 +34,13 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     async_add_entities(entities)
 
 
-class SpeedtestSensor(RestoreEntity):
+class SpeedtestSensor(CoordinatorEntity, RestoreEntity):
```

Comment:
Coordinator does not call `super().async_added_to_hass()`, so this probably breaks right? Opened https://github.com/home-assistant/core/pull/39416 to fix it.